### PR TITLE
added-feat-design-menupage-contactpage

### PR DESCRIPTION
**Pull request by AlinaHauriluk**
added new-dishes component of menu page, contact-info, contact-map in contact page, corrected import, added eslint-plugin-import

### CODE REVIEW

File: client/src/components/chefs-info/index.tsx [hunk 1]
```diff
@@ -20,11 +15,11 @@ const ChefsInfo: React.FC <ChefsInfoProps> = ({chefs}) => {
   return (
     <>
       {chefs.map(chef => (
-        <ChefsInfoWrapper data-aos="fade-down" key={chef.id}>
-          <ChefsInfoPhoto photo={chef.photo} />
-          <ChefsInfoName>{chef.name}</ChefsInfoName>
-          <ChefsInfoPosition>{chef.position}</ChefsInfoPosition>
-        </ChefsInfoWrapper>
+        <_.ChefsInfoWrapper data-aos="fade-down" key={chef.id}>
```
Comment (tatsianamatesha) [inline client/src/components/chefs-info/index.tsx]: why do you need `_.`? it's a bad practise

File: client/src/components/contact-map/index.tsx [hunk 2]
```diff
@@ -0,0 +1,32 @@
...
+import AOS from "aos";
+import "aos/dist/aos.css";
+
+
+const ContactMap = () => {
+  useEffect(() => {
+    AOS.init({ duration: 2000 });
+  }, []);
+
+  const coordinates = [35.1815784, -120.7474753];
```
Comment (tatsianamatesha) [inline client/src/components/contact-map/index.tsx]: move this constant above component. we don't need initializate this on every render.

Comment (AlinaHauriluk) [reply client/src/components/contact-map/index.tsx]: thank you i corrected

File: client/src/components/chefs-info/index.tsx (hunk 1 above)
Comment (AlinaHauriluk) [reply client/src/components/chefs-info/index.tsx]: i corrected it

File: client/src/components/chefs-info/index.tsx [hunk 3]
```diff
@@ -1,25 +1,30 @@
 import React, { useEffect } from "react";
```
Comment (vladimirtopolev) [inline client/src/components/chefs-info/index.tsx:1]: it's better to import FC as the following import now:
`import type {FC} from 'react'`

File: client/src/components/contact-map/index.tsx [hunk 4]
```diff
@@ -5,14 +5,13 @@ import { Container } from "../../box/absolute-box";
 import AOS from "aos";
 import "aos/dist/aos.css";
 
+const coordinates = [35.1815784, -120.7474753];
```
Comment (vladimirtopolev) [inline client/src/components/contact-map/index.tsx:8]: Let's create a separate file in folder `config` and define some `constants.ts` file where you may put all such hard-coded values.

File: client/src/components/menu-list/index.tsx [hunk 5]
```diff
@@ -60,39 +73,39 @@ const MenuList: React.FC<MenuProps> = ({ title, subtitle }) => {
...
+          </MenuWrapper>
         )}
 
         <FlexJustifyCenter>
           {data.map((item) => (
-            <_.MenuPart key={item.id} data-aos="fade-down">
-              <_.MenuPartTitle>{item.title}</_.MenuPartTitle>
+            <MenuPart key={item.id} data-aos="fade-down">
+              <MenuPartTitle>{item.title}</MenuPartTitle>
               {item.dishes.map((i) => (
```
Comment (vladimirtopolev) [inline client/src/components/menu-list/index.tsx:92]: name `i` here is terrible name

File: client/src/components/new-experience/index.tsx [hunk 6]
```diff
@@ -1,38 +1,45 @@
...
 
   return (
-    <_.SectionNewExperience>
+    <SectionNewExperience>
       <FlexContainer data-aos="fade-down">
         <Flex>
-          <_.NewExperienceImage />
-          <_.NewExperienceImageTwo />
+          <NewExperienceImage />
+          <NewExperienceImageTwo />
```
Comment (vladimirtopolev) [inline client/src/components/new-experience/index.tsx]: Well, could we combine `NewExperinceImage` and `NewExperienceImageTwo` in one component and change logic passing appropriate prop which would switching between Component representation?

**Pull request merged**